contour/views.py

```diff
@@ -1104,3 +1109,95 @@ def get_info_from_reader(reader):
         var_symbols.append(reader[i])
         i += 1
     return var_names, var_symbols
+
+
+def sort_plotted_figures(probabilistic_model):
+    """
+    Sorts the fit images of a probabilistic model to generate clear overview
+    in the template.
+
+    Parameters
+    ----------
+    probabilistic_model : ProbabilisticModel
+        A probabilistic model generated by a fit.
+
+    Return
+    -----
+    srt_figures : list of TemplateImages
+        The TemplateImages represents all images that linked to a parameter of
+        a distribution.
+    """
+    srt_figures = []
+
+    dist_models = DistributionModel.objects.filter(
+        probabilistic_model=probabilistic_model)
+
+    for i, dist in enumerate(dist_models):
+        plotted_figures = PlottedFigure.objects.filter(
+            distribution_model=dist)
+
+        if len(plotted_figures) == 1:
+            temp_image = TemplateImage()
+            temp_image.var_number = i + 1
+            temp_image.param_image = plotted_figures[0]
+            temp_image.param_name = 'independent'
+            srt_figures.append(temp_image)
+        else:
+            param_images = []
+            pdf_images = []
+            for j, plotted_figure in enumerate(plotted_figures):
+                if plotted_figure.parameter_model is not None or 'None' in plotted_figure.image.url:
+                    param_images.append(plotted_figure)
+                else:
+                    pdf_images.append(plotted_figure)
+
+            for param in param_images:
+                temp_image = TemplateImage()
+                temp_image.var_number = i + 1
+                temp_image.param_image = param
+                if 'None' not in param.image.url:
+                    temp_image.pdf_images = pdf_images
+                    if dist.distribution == 'Weibull':
```

Comment:
Use the variable names that can be found in pdf equation (e.g. lambda symbol, theta symbol, ...)